packages/cli/src/lib/missing-routes.ts

```diff
@@ -30,12 +30,16 @@ const REQUIRED_ROUTES = [
 
   'account',
   'account/login',
-  'account/register',
   // 'account/addresses',
   // 'account/orders',
   'account/orders/:orderId',
-  'account/reset/:id/:token',
-  'account/activate/:id/:token',
+  // -- Added for CAAPI:
+  'account/authorize',
```

Comment:
Hmmm, does this mean the `authUrl` option to `customerClient` should default to `/account/authorize`?.

Reply:
mm......ya maybe